build/data/services/service_data.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Can we move this file to `data` directory instead of the `build/data`?

Reply:
Done.